URL Detection

```diff
@@ -0,0 +1,3 @@
+NetworkCommunicationEvents  
+| where RemoteUrl == "www.microsoft.com"
```

Comment:
Usually we add a time filtering in the query
`| where EventTime > ago(7d)`
to make it obvious for users that they will get the best performance when filtering on shorter time priods